README.md

```diff
@@ -24,12 +24,15 @@ National Cybersecurity R&D Laboratory (NCL)\
 National University of Singapore (NUS)\
 Singapore
 
-Maintainer : Aris Cahyadi Risdianto - ariscr@ncl.sg
- 
-Supervisor : Prof. Ee-chien Chang - changec@comp.nus.edu.sg
+Project Leader: Aris Cahyadi Risdianto - ariscr@ncl.sg
+Project Investigator (PI) : Prof. Ee-Chien Chang - changec@comp.nus.edu.sg
```

Comment:
Should this be `Principal Investigator (PI)` instead of `Project Investigator (PI)`?